apps/ledger-live-mobile/ios/ledgerlivemobile/Info.plist

```diff
@@ -35,7 +35,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.40.1</string>
+	<string>3.47.2</string>
```

Comment:
Not sure you need to touch this either because it's handle by scripts during the deploy on main/prod